index.html

```diff
@@ -39,108 +45,156 @@ <h1><span class="first-part">I'm Tonny Tei,</span><br><span class="second_part">
 
     <section id="mid">
         <!-- second page -->
-         <div class="left-block">
-            <div class="primary-image">
-                <img src="images/project1.png" alt="photo_of_my_projects">
-            </div>
-            <div class="title">
-                <h1>Tonic</h1>
+         <div class="left-block" id="second-page">
+            <div class="primary-image" id="grid-one">
+                <img src="images/project1.png" alt="photo_of_my_projects" class="for-small-size">
+                <img src="images/Snapshoot Portfolio.png" alt="Snapshoot_portfolio" class="for-big-size">
             </div>
-            <div class="frame-2">
-                <ul>
-                    <li>CANOPY</li>
-                    <li>Back End Dev</li>
-                    <li>2015</li>
-                </ul>
-            </div>
-            <p class="primary-text"> Daily selection of privately personalised reads" no accounts or signups required. </p>
-            <div class="tags"> 
-                <ul>
-                    <li>html</li>
-                    <li>CSS</li>
-                    <li>javascript</li>
-                </ul>
+
+            <div id="grid-two" class="left">
+                <div class="title">
+                    <h1>Tonic</h1>
+                </div>
+                <div class="frame-2">
+                    <ul>
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <p class="primary-text"> Daily selection of privately personalised reads" no accounts or signups required. </p>
+                <div class="tags"> 
+                    <ul>
+                        <li>html</li>
+                        <li>CSS</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="action"> <a href="#">See project</a></div>
```

Comment:
- [ ] Kindly ensure that you reduce the border width of all your project card button to match your chosen Figma design 👍🏽 . Kindly follow the design specification on Figma 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/194625190-c25d907b-c686-4b8e-bffc-acdad58c7cb1.png)  | ![image](https://user-images.githubusercontent.com/63473584/194625246-80e8dfa6-c6ac-4a15-8ae2-758a3330d8c4.png)  |